crates/rethnet_evm/benches/state.rs

```diff
@@ -61,5 +155,13 @@ fn bench_storage(c: &mut Criterion) {
     );
 }
 
-criterion_group!(benches, bench_basic, bench_code_by_hash, bench_storage);
+criterion_group!(
+    benches,
+    /*
+    bench_basic,
+    bench_code_by_hash,
+    bench_storage,
+    */
```

Comment:
I think you committed this accidentally?

Reply:
Yes! Thank you! Addressed in https://github.com/NomicFoundation/hardhat/pull/3953/commits/357a0f3f0199c50161fa8a629f995dfadcc712e7